runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java

```diff
@@ -219,6 +230,99 @@ public void onWidgetLoaded(WidgetLoaded event)
 		}
 	}
 
+	@Subscribe
+	public void itemContainerChanged(ItemContainerChanged event)
+	{
+		final ItemContainer container = event.getItemContainer();
+		observeForWintertodtLootEntry(container);
+	}
+
+	private void observeForWintertodtLootEntry(ItemContainer container)
+	{
+		if (container != client.getItemContainer(InventoryID.INVENTORY))
+		{
+			return;
+		}
+
+		final Item[] items = container.getItems();
+
+		// EXTRA_SUPPLY_CRATE is the reward when you trade in items to Ignisia
+		long numberOfSupplyCrates = countSupplyCrates(items);
+
+		HashMultiset<Integer> currentSnapshot = generateInventorySnapshot(items);
+
+		// We want to make sure that we actually opened the supply crate, not banked it
+		if (numberOfSupplyCrates != lastNumberOfSupplyCrates && inventorySnapshot != null)
+		{
+			log.debug("Number of supply crates in inventory has changed from {} to {}", lastNumberOfSupplyCrates, numberOfSupplyCrates);
+
+			// we lost a crate by opening it
+			if (numberOfSupplyCrates < lastNumberOfSupplyCrates && !bankWidgetIsOpen() && !depositWidgetIsOpen())
+			{
+				List<ItemStack> loot = generateSnapshotDelta(currentSnapshot, inventorySnapshot);
+				final LootTrackerItem[] entries = buildEntries(stack(loot));
+				SwingUtilities.invokeLater(() -> panel.add(WINTERTODT_EVENT_TYPE, -1, entries));
+			}
+
+			lastNumberOfSupplyCrates = numberOfSupplyCrates;
+		}
+
+		inventorySnapshot = currentSnapshot;
+	}
+
+	private int countSupplyCrates(Item[] inventory)
+	{
+		int count = 0;
+
+		for (Item item : inventory)
+		{
+			if (item.getId() == ItemID.SUPPLY_CRATE || item.getId() == ItemID.EXTRA_SUPPLY_CRATE)
+			{
+				count = count + 1;
+			}
+		}
+
+		return count;
+	}
+
+	private HashMultiset<Integer> generateInventorySnapshot(Item[] inventory)
+	{
+		HashMultiset<Integer> snapshot = HashMultiset.create();
+
+		for (Item item : inventory)
+		{
+			if (WintertodtLoot.isWintertodtLoot(itemManager.canonicalize(item.getId())))
+			{
+				snapshot.add(item.getId(), item.getQuantity());
+			}
+		}
+
+		return snapshot;
+	}
+
+	private List<ItemStack> generateSnapshotDelta(	HashMultiset<Integer> newItems,
```

Comment:
Formatting has gone whacky here

Reply:
The formatting issue is that `generateSnapshotDelta(\tHashMultiSet` should be `generateSnapshotDelta(HashMultiSet`